test/e2e/quadlet/device.volume

```diff
@@ -0,0 +1,13 @@
+## assert-key-contains Service ExecStart " --opt o=uid=0,gid=11,rw,compress=zstd "
+## assert-key-contains Service ExecStart " --opt type=btrfs "
+## assert-key-contains Service ExecStart " --opt device=/dev/vda1 "
+## assert-key-contains Service ExecStart " --opt nocopy "
+
+[Volume]
+# Test usernames too
+User=root
+Group=11
+Device=/dev/vda1
+Type=btrfs
+Options=rw,compress=zstd
+Copy=no
```

Comment:
Could you add one test with `Copy=yes`?  I am a bit puzzled by the "yes" and "no" vs "true" and "false".  Is that intentional/conventional?

Reply:
I've added the test.
AFAIK the convention in systemd is `yes/no`. The same is used in other tests